src/unstructured_client/_hooks/custom/clean_server_url_hook.py

```diff
@@ -0,0 +1,34 @@
+from typing import Tuple
+from urllib.parse import ParseResult, urlparse, urlunparse
+
+import requests
+
+from ..types import SDKInitHook
```

Comment:
It's better to avoid using relative imports..

Reply:
Yep, forgot to change that. Fixed.